src/Take.Blip.Builder/Models/Input.cs

```diff
@@ -10,7 +11,8 @@ namespace Take.Blip.Builder.Models
     /// </summary>
     public class Input : IValidable
     {
-        private static readonly Regex VariableValidationRegex = new Regex("^([a-zA-Z0-9\\.]+)$", RegexOptions.Compiled);
+        private static IConfiguration _configuration = new ConventionsConfiguration();
+        private static readonly Regex VariableValidationRegex = new Regex("^([a-zA-Z0-9\\.]+)$", RegexOptions.Compiled, _configuration.RegexTimeout);
```

Comment:
This regex it's simple and compiled, its okay to not add a timeout

Reply:
Fixed. If the Regex it's simple, I will delete the IConfiguration in the start of the code ( leaving whitout timeout )